### ISSUE TITLE
Payment details should contain issue and PR numbers

### ISSUE THREAD

**Issue by krzyk** [zerocracy/farm#1556]
Right now payment details contain amounts, which is pointless, it should contain issue and PR numbers.
E.g. for Paypal there is 1000 characters limit, which should suffice for most cases.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by krzyk** [comment]
@g4s8 It is really hard to keep track of what was paid and what was missed.

**Comment by g4s8** [comment]
@krzyk are you suggesting to include all debts information to Paypal messages? I don't think it's possible to include job numbers here, because 0crat may send payment not for job, but for release, or tution fee or maybe for something else.

**Comment by krzyk** [comment]
@g4s8 Yeah, but majority of the payments would probably be for jobs, other payments can be initially added as "none" + payment amount and when it is possible we could write the source, so release or tutition fee.


The current payment message is completely useless, it contains only amounts of payments and is truncated after 100 chars I think or so.

**Comment by g4s8** [comment]
@krzyk so how about using debt's details as message for paypal?
```
Order was finished, quality is "good" at gh:zerocracy/farm#1
Order was finished, quality is "good" at gh:zerocracy/farm#2
Order was finished, quality is "good" at gh:zerocracy/farm#3
```

**Comment by krzyk** [comment]
@g4s8 This will fit 12 REV jobs in zerocracy, but should at least 16 REV jobs.
If we could get rid of the `"Order was finished, "` at the beginning it would be perfect.
Or maybe just regexp for "((gh|gl):[a-zA-Z0-9-]+#[0-9]+)" - (github repo pattern should be better) if it matches, then get this and add to message, if it doesn't match, then add the message as is (so it would cover arc bonus and student fees).

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)
